09-features.md

```diff
@@ -42,7 +42,7 @@ The Context column decodes as follows:
 | 16/17 | `basic_mpp`                       | Node can receive basic multi-part payments                | IN9      | `payment_secret`          | [BOLT #4][bolt04-mpp]                                                 |
 | 18/19 | `option_support_large_channel`    | Can create large channels                                 | IN       |                           | [BOLT #2](02-peer-protocol.md#the-open_channel-message)               |
 | 22/23 | `option_anchors`                  | Anchor commitment type with zero fee HTLC transactions    | IN       |                           | [BOLT #3][bolt03-htlc-tx], [lightning-dev][ml-sighash-single-harmful] |
-| 24/25 | `option_route_blinding`           | Node supports blinded paths                               | IN9      |                           | [BOLT #4](bolt04-route-blinding)                                      |
+| 24/25 | `option_route_blinding`           | Node supports blinded paths                               | IN9      |                           | [BOLT #4](04-onion-routing.md#route-blinding)                                      |
```

Comment:
Woops my bad, good catch, I used parenthesis instead of brackets. I think that since the link is defined below, the simpler fix is:

```suggestion
| 24/25 | `option_route_blinding`           | Node supports blinded paths                               | IN9      |                           | [BOLT #4][bolt04-route-blinding]                                      |
```